### PR TITLE
Create tests for the app

### PR DESCRIPTION
**Pull request by marinawangmw**
#### 📄&nbsp; Description:

Created snapshot testing and Enzyme ui testing  for the app

---

#### ⏯&nbsp; Demo:

Screen Shot 2021-09-28 at 14 56 04

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+API_BASE_URL=‘https://api.themoviedb.org/3’
+API_KEY=‘ca02733a9fd4f7fa59475a18cac22804’
+IMAGE_API_BASE_URL=‘https://image.tmdb.org/t/p/w500’
+BUILD_VARIANT=‘DEV’
```
Comment (francoSimeone) [inline .env]: we should not push env variables to github

File: jest.setup.js [hunk 2]
```diff
@@ -35,4 +38,53 @@ jest.mock('react-native-config', () => ({
...
+// };
+// global.requestAnimationFrame = function (callback) {
+//   return setTimeout(callback, 0);
+// };
+// global.cancelAnimationFrame = function (id) {
+//   clearTimeout(id);
+// };
+
+// copyProps(window, global);
+
```
Comment (francoSimeone) [inline jest.setup.js]: remove commented code

File: src/screens/MovieCollection/MovieCollection.test.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import { shallow } from 'enzyme';
+import renderer from 'react-test-renderer';
+import { render } from '@testing-library/react-native';
+import { withProviders } from '@/test-utils';
+import { MovieCollection } from '@/screens';
+import { movieList, navigation } from '@/test-utils/helper';
+import { MovieItem } from '@/components';
+
+const emptyRoute = { params: { collection: [{ movieList: [], title: 'test empty' }] } };
+const routeWithData = {
```
Comment (francoSimeone) [inline src/screens/MovieCollection/MovieCollection.test.js:11]: are this names correct? or should they be inverted?

Comment (marinawangmw) [reply src/screens/MovieCollection/MovieCollection.test.js:11]: The names were right, the data they contain is wrong 😂

Comment (francoSimeone) [reply src/screens/MovieCollection/MovieCollection.test.js:11]: 😂

**Review by francoSimeone — APPROVED** [review]

**Pull request merged**